bundles/org.openhab.binding.electroluxappliances/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,472 @@
+		<label>Humidity</label>
+		<description>The measured humidity.</description>
+		<category>Humidity</category>
+		<state readOnly="true" min="0" max="100" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="tvoc">
+		<item-type>Number:Dimensionless</item-type>
+		<label>TVOC</label>
+		<description>The total Volatile Organic Compounds.</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+
+	<channel-type id="pm1">
+		<item-type>Number:Density</item-type>
+		<label>PM1</label>
+		<description>The Particulate Matter 1 (0.001mm).</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="pm2_5">
+		<item-type>Number:Density</item-type>
+		<label>PM2.5</label>
+		<description>The Particulate Matter 2.5 (0.0025mm).</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="pm10">
+		<item-type>Number:Density</item-type>
+		<label>PM10</label>
+		<description>The Particulate Matter 10 (0.01mm).</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="co2">
+		<item-type>Number:Dimensionless</item-type>
+		<label>CO2</label>
+		<description>The measured CarbonDioxide.</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="filterLife">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Filter Life</label>
+		<description>The filter life indication.</description>
+		<state readOnly="true" min="0" max="100" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="doorOpen">
+		<item-type>Contact</item-type>
+		<label>Door Status</label>
+		<description>The door status Open/Closed.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="fanSpeed">
+		<item-type>Number</item-type>
+		<label>Fan Speed Setting</label>
+		<description>The fan speed setting.</description>
+		<state>
+			<options>
+				<option value="1">Level 1</option>
+				<option value="2">Level 2</option>
+				<option value="3">Level 3</option>
+				<option value="4">Level 4</option>
+				<option value="5">Level 5</option>
+				<option value="6">Level 6</option>
+				<option value="7">Level 7</option>
+				<option value="8">Level 8</option>
+				<option value="9">Level 9</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="workMode">
+		<item-type>String</item-type>
+		<label>Work Mode Setting</label>
+		<description>The work mode setting.</description>
+		<state>
+			<options>
+				<option value="PowerOff">Power Off</option>
+				<option value="Auto">Automatic</option>
+				<option value="Manual">Manual</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="ionizer">
+		<item-type>Switch</item-type>
+		<label>Ionizer Status</label>
+		<description>The ionizer status.</description>
+	</channel-type>
+
+	<channel-type id="uiLight">
+		<item-type>Switch</item-type>
+		<label>UI Light</label>
```

Comment:
What does UI stand for in this specific case?
And what does it do, does this switch signal alarm, does it switch the light/on off?  And is this read-only?

Reply:
Maybe better to adjust it to `Display Light` for clarity.

Reply:
I can change the label to Display Light, but most channel names are reflecting the name used in the api, you could argue that other channels should have more understandable names so I rather keep the channel name as it is.